### DISCUSSION TITLE
Unified search for templates, template partials, and template variables

### DISCUSSION THREAD

**Discussion by omejelle** [ExpressionEngine/ExpressionEngine#3258] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
when searching in the templates section in the cp the upper search box does not search in the template partials or the template variables. There are separate search boxes for those sections, but seems a bit clumsy

**Describe the solution you'd like**
Have 1 searchbox that searches in the entire template section, so in templates, template variables and template partials

**Describe alternatives you've considered**
a search version such as used on devot-ee (where you can select where to search in)

**Teachability, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this and possibly write out a version the docs. Some of the best features are born from writing documentation first. Consider making a documentation pull request from a feature branch here: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by PaulBaileyAtChan** [comment]
Second this feature request. There's no obvious benefit to forcing separate searches for templates, template partials, and template variables, and it multiplies the time/steps required to search all templates by 3. A single unified template search could/should be able to search all template types. Results of a unified template search could still be marked by template type, for clarity.
